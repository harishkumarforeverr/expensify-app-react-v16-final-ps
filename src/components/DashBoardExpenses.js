import React from "react"
import ExpenseList from "./ExpenseList";
import ExpenseListFilter from "./ExpenseListFilter";
import ExpenseSummary from "./ExpenseSummary";

const DashBoardExpenses=()=>{
    return (
        <div> 
         <ExpenseSummary />
         <ExpenseListFilter />
         <ExpenseList />
        </div>
    )
}
export default DashBoardExpenses;