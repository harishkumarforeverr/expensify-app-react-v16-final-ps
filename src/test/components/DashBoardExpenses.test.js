import React from 'react';
import {shallow} from "enzyme";
import DashBoardExpenses from "../../components/DashBoardExpenses";

test('import DashBoardExpenses from "../../components/DashBoardExpenses"',()=>{
    const wrapper=shallow(<DashBoardExpenses />);
    expect(wrapper).toMatchSnapshot();
})
