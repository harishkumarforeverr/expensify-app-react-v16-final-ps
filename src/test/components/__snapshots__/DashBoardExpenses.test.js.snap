// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import DashBoardExpenses from "../../components/DashBoardExpenses" 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
